src/library/mapper/JsonToWeatherMapper.ts

```diff
@@ -0,0 +1,58 @@
+import {JsonToWeatherMapperInterface} from "./jsonToWeatherMapperInterface";
+import {WeatherInterface} from "./../models/WeatherInterface";
+import {WeatherFactoryInterface} from "./../factory/WeatherFactoryInterface";
+
+export class JsonToWeatherMapper implements JsonToWeatherMapperInterface {
+
+    public static TEMPERATURE_CONSTANT: number = 30;
```

Comment:
This is not required - needs to remove - after reading the forecast.io api doc, which is pretty terrible! API's documentation should really have a great DX. Apiary is a great one. https://apiary.io/